Dockerfile.webhook

```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/k8snetworkplumbingwg/sriov-network-operator
 COPY . .
 RUN make _build-webhook BIN_PATH=build/_output/cmd
 
-FROM quay.io/centos/centos:stream9
+FROM ${BASE_IMAGE_GO_DISTROLESS_DEV:-nvcr.io/nvidia/distroless/go:v3.2.1-dev}
```

Comment:
we should update only `.nvidia` Docker files

Reply:
my bad, will fix it asap